ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -552,11 +662,12 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Release.xcconfig */;
 			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
```

Comment:
👀

Reply:
How to handle these new code signings? Automatic seems to be sexy, but other devs may need to auto generate identities, too. (But will not change this setting) 
On the other hand, can fastlane handle this? What do you think? Will it build under CI now?

Reply:
Okay, let's hope so. I just don't want to get into the habit of committing continuous changes to it.